lib/PaymentMethod.php

```diff
@@ -26,7 +26,7 @@
  * @property \Stripe\StripeObject $bancontact
  * @property \Stripe\StripeObject $billing_details
  * @property \Stripe\StripeObject $boleto
- * @property \Stripe\StripeObject $card
+ * @property null|\Stripe\Card $card If this is a card PaymentMethod, this hash contains the user’s card details.
```

Comment:
@arouze This change is incorrect. The `Card` class is entirely separate in the stripe-php library and represents an older resource in the API, the [Card object](https://stripe.com/docs/api/cards/object) for a different API. While they share some properties, they are unrelated and this change wouldn't be something we could merge.